### PR TITLE
fix bug when rewriting deprecated unittest functions

### LINKED ISSUE #749: unittest.makeSuite() with prefix incorrectly rewritten
```py
return unittest.makeSuite(Test, 'test')
```

is rewritten to:

```py
return unittest.defaultTestLoader.loadTestsFromTestCase(Test, 'test')
```

which is invalid:

```
    return unittest.defaultTestLoader.loadTestsFromTestCase(Test, 'test')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: TestLoader.loadTestsFromTestCase() takes 2 positional arguments but 3 were given
```

This works:
```py
return unittest.defaultTestLoader.loadTestsFromTestCase(Test)
```

### PR DESCRIPTION
**Pull request by piazzesiNiccolo**
Closes #749.

I saw that #759 was closed without actually fixing the issue.

### CODE REVIEW

File: pyupgrade/_plugins/unittest_aliases.py [hunk 1]
```diff
@@ -56,10 +57,13 @@ def visit_Call(
         )
         yield ast_to_offset(node.func), func
     elif (
-            isinstance(node.func, ast.Attribute) and
-            isinstance(node.func.value, ast.Name) and
-            node.func.value.id == 'unittest' and
-            node.func.attr in FUNCTION_MAPPING
```
Comment (asottile) [inline pyupgrade/_plugins/unittest_aliases.py:62]: don't touch unrelated code to imbue your style on it

Comment (piazzesiNiccolo) [reply pyupgrade/_plugins/unittest_aliases.py:62]: oops, left formatter settings on, sorry

File: tests/features/unittest_aliases_test.py [hunk 2]
```diff
@@ -7,14 +7,17 @@
...
     (
         pytest.param(
             'class ExampleTests:\n'
             '    def test_something(self):\n'
             '        self.assertEqual(1, 1)\n',
             id='not a deprecated alias',
         ),
+        # https://github.com/asottile/pyupgrade/issues/749#
+        ('unittest.makeSuite(Tests, "arg")'),
+        ('unittest.makeSuite(Tests, prefix="arg")'),
```
Comment (asottile) [inline tests/features/unittest_aliases_test.py:20]: these parens do nothing

File: tests/features/unittest_aliases_test.py [hunk 3]
```diff
@@ -7,14 +7,17 @@
...
-    ('s',),
+    's',
     (
         pytest.param(
             'class ExampleTests:\n'
             '    def test_something(self):\n'
             '        self.assertEqual(1, 1)\n',
             id='not a deprecated alias',
         ),
+        # https://github.com/asottile/pyupgrade/issues/749#
```
Comment (asottile) [inline tests/features/unittest_aliases_test.py:18]: if this link is important put it in the pytest id -- if it's not remove it -- also it contains a typo

**Comment by asottile** [comment]
the commit message is kinda unclear here -- it probably should be something closer to "prevent rewriting unittest module functions with additional arguments" -- or something like it

**Comment by piazzesiNiccolo** [comment]
just mistakenly copied an old git command when changing commit message, fixing this

**Review by asottile — APPROVED** [review]
